src/lib/snack-bar/snack-bar.ts

```diff
@@ -145,6 +148,19 @@ export class MatSnackBar {
     // We can't pass this via the injector, because the injector is created earlier.
     snackBarRef.instance = contentRef.instance;
 
+    // Subscribe to the breakpoint observer and attach the mat-snack-bar-handset class as
+    // appropriate. This class is applied to the overlay element because the overlay must expand to
+    // fill the width of the screen for full width snackbars.
+    RxChain.from(this._breakpointObserver.observe(Breakpoints.Handset))
+      .call(takeUntil, RxChain.from(overlayRef.detachments()).call(first).result())
```

Comment:
For a single operator you don't really need the `RxChain`, you can just `.call` it directly. E.g. `first.call(overlayRef.detachments())`.

Reply:
I did find that frustrating to have it in `RxChain` again, not sure why I didn't think to look at calling it this way.